apps/src/code-studio/progressRedux.js

```diff
@@ -262,6 +265,13 @@ export default function reducer(state = initialState, action) {
     };
   }
 
+  if (action.type === SET_RELOAD_TEACHER_PANEL_PROGRESS) {
```

Comment:
Could you elaborate a bit on why we need this value in redux?  Is there a scenario where we need to re-render but redux-managed state doesn't change?  (Usually, we just update the underlying data and the appropriate components re-renders, right?)

Reply:
Right - we're not storing the teacher panel bubble data in redux, it's stored in the TeacherPanel component state, since that's the only component that needs to know about the data. However when a teacher makes a change in the feedback component we need to communicate to the TeacherPanel component that it needs to refresh it's data so that's what's happening here.

Reply:
Ah, ok, I think I understand now.  Do we use this pattern anywhere else?  (This feels potentially simpler than putting the data in redux but it also feels a bit non-standard.)  Let's chat tomorrow.